stp_zmq/zstack.py

```diff
@@ -946,6 +955,20 @@ def prepare_to_send(self, msg: Any):
         self.msgLenVal.validate(msg_bytes)
         return msg_bytes
 
+    @staticmethod
+    def get_monitor_events(monitor_socket, non_block=True):
```

Comment:
1) There is a similar method in Remote. Should we get rid of duplication?
2) There is `events.append(message['event'])` line in Remote's implementation. Should it be the same?

Reply:
I've added a comment for this method. My concern is that working with monitor socket is not fully correct and should be reviewed. Now we call get_monitor_socket() each time when we want to get it, it's strange. If it is a singleton then it is ok, but I'm not sure. So I'll create a separate ticket and clean up this implementation.